scalingbitcoin/tokyo-2018/atomic-swaps.md

```diff
@@ -3,7 +3,7 @@ title: Atomic Swaps
 transcript_by: Bryan Bishop
 categories: ['conference']
 tags: ['atomic swaps']
-speakers: ['Thomas Eizinger', 'Lloyd Fournier', 'Phillip Hoenisch']
```

Comment:
why were these other speakers removed?

Reply:
It seems that I've skipped verification on this. I was wrong
![Screenshot 2022-11-18 at 10 42 35 PM](https://user-images.githubusercontent.com/18506343/202798968-337cca59-eec7-4a83-bdce-c54c5fce706e.png)
[src](https://tokyo2018.scalingbitcoin.org/)